packages/migrate/src/commands/MigrateDiff.ts

```diff
@@ -144,6 +153,8 @@ ${chalk.bold('Examples')}
       return this.help(args.message)
     }
 
+    await checkUnsupportedDataProxy('migrate diff', args, false)
```

Comment:
Migrate diff does not implicitly load the schema at it's default location (it needs --from and --to always), so we apply this to our logic too.